demo/config.js

```diff
@@ -0,0 +1,15 @@
+//Config file
+//TODO: Logic should be separated from configuration
+
+var config_data = {
+  'GENERAL_CONFIG': {
+    'END_POINT_BASE_URL': 'http://localhost:8079/blazegraph/'
```

Comment:
I'm conflicted whether the keys of a constant map also need to be all caps. Maybe only the variable/constant name? Let's discuss this at some point. (not right now)